.github/workflows/publish-npm-package.yml

```
@@ -63,3 +69,66 @@ jobs:
+      - name: Create Git Tag
+        if: steps.check_stable.outputs.is_stable == 'true'
+        run: |
+          VERSION="v${{ github.event.inputs.version }}"
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+          git tag -a $VERSION -m "Release $VERSION"
+          git push origin $VERSION
+
+      - name: Create GitHub Release
+        if: steps.check_stable.outputs.is_stable == 'true'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)